src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py

```diff
@@ -818,6 +820,39 @@ def test_containerapp_managed_service_binding_e2e(self, resource_group):
             JMESPathCheck('length(properties.template.containers[0].env[?name==`AZURE_MYSQL_HOST`])', 1),
             JMESPathCheck('length(properties.template.containers[0].env[?name==`AZURE_POSTGRESQL_HOST`])', 1)
         ])
+        self.cmd('containerapp update -g {} -n {} --bind kafkaconfluent,bootstrap_server={},kafka_key={},kafka_secret={}'.format(
```

Comment:
Please check if the `properties.provisioningState` is `Succeeded`.
Same as below following create/update command.